universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -119,7 +116,8 @@ private ContainerTag renderLocaleNotSupportedToast(
 
   private Tag renderBlockWithSubmitForm(Params params) {
     if (params.block().isFileUpload()) {
-      return renderFileUploadBlockSubmitForms(params);
+      return fileUploadStrategy.renderFileUploadBlockSubmitForms(
+          params, applicantQuestionRendererFactory);
```

Comment:
suggestion (blocking): looks like `applicantQuestionRendererFactory` should be injected in `FileUploadStrategy` rather than passed in here as an arg?

Reply:
I thought this would be weird because fixing one of the above comments required injecting FileUploadViewStrategy into ApplicantQuestionRendererFactory. Could you let me know what you think about the current iteration?

Reply:
Looks good